src/main/java/de/mediathekview/mserver/crawler/wdr/tasks/WdrFilmDetailTask.java

```diff
@@ -0,0 +1,59 @@
+package de.mediathekview.mserver.crawler.wdr.tasks;
+
+import de.mediathekview.mlib.daten.Film;
+import de.mediathekview.mserver.base.utils.UrlUtils;
+import de.mediathekview.mserver.crawler.basic.AbstractCrawler;
+import de.mediathekview.mserver.crawler.basic.AbstractDocumentTask;
+import de.mediathekview.mserver.crawler.basic.AbstractUrlTask;
+import de.mediathekview.mserver.crawler.basic.TopicUrlDTO;
+import de.mediathekview.mserver.crawler.wdr.parser.WdrFilmDeserializer;
+import de.mediathekview.mserver.crawler.wdr.parser.WdrFilmPartDeserializer;
+import java.util.Optional;
+import java.util.Set;
+import java.util.concurrent.ConcurrentLinkedQueue;
+import org.jsoup.nodes.Document;
+
+public class WdrFilmDetailTask extends AbstractDocumentTask<Film, TopicUrlDTO>  {
+
+  public WdrFilmDetailTask(AbstractCrawler aCrawler, ConcurrentLinkedQueue<TopicUrlDTO> aUrlToCrawlDtos) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The type of the "aUrlToCrawlDtos" object should be an interface such as "Queue" rather than the implementation "ConcurrentLinkedQueue". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)